### PR TITLE
GVT: Hardcode issue in "apic lb -h"

### PR DESCRIPTION
**Pull request by dhmlau**
Overwrite the default help messages from Yeoman for
- --help
- --skip-cache
- --skip-install

connected to: https://github.com/strongloop/generator-loopback/issues/331

### CODE REVIEW

**Comment by dhmlau** [comment]
@bajtos , I found another way to update the option description based on the option names, in the function in here: https://github.com/strongloop/generator-loopback/blob/master/lib/help.js#L101-L128.  But thought that it's more hacky that way.  
If that's the preferred place to make the change, I'm good with that too.  Please advise.

**Comment by bajtos** [comment]
>  I found another way to update the option description based on the option names, in the function in here: https://github.com/strongloop/generator-loopback/blob/master/lib/help.js#L101-L128. But thought that it's more hacky that way.

I think that may not work as intended. When an option is defined with a translated description, then `optionsBuild` will try to translate it again, won't it?

**Comment by raymondfeng** [comment]
@dhmlau Instead of overriding `option()`, we can delete the built-in options before adding them back with translated description. For example,

```
delete this._options['help'];
delete this._options['skip-cache'];
delete this._options['skip-install'];
```

Or we can add a method:

```
overrideOption(name, value) {
  delete this._options[name];
  this.option(name, value);
}

this.overrideOption('help', ...);
```

**Comment by dhmlau** [comment]
CI failed for `loopback-cli` because the message in the [output](https://github.com/strongloop/loopback-cli/blob/master/test/fixtures/help-lb.txt) has changed due to this PR.

**Comment by dhmlau** [comment]
@raymondfeng @bajtos , thanks for your input.  I've made the change as suggested.  Please review again.  Thanks!

File: app/index.js [hunk 1]
```diff
@@ -29,9 +29,28 @@ module.exports = yeoman.Base.extend({
       type: String,
     });
 
+    // delete the default options coming from Yeoman
+    delete this._options['help'];
```
Comment (raymondfeng) [inline app/index.js]: It will be nice to add a comment explaining why these options need to be deleted 1st

**Comment by bajtos** [comment]
> CI failed for loopback-cli because the message in the output has changed due to this PR.

@dhmlau Please open a follow-up pull request to fix loopback-cli as soon as this patch is landed.

**Comment by dhmlau** [comment]
@bajtos , you're right!!  Let me modify my PR.

**Pull request merged**